foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/ClientClosedEvent.java

```diff
@@ -17,6 +17,9 @@
 
 package org.apache.servicecomb.foundation.vertx;
 
+/**
```

Comment:
indentation is one space?
also need to check other codes.

Reply:
I copy before, and I am using GoogleStyle in etc

Reply:
sorry, i made a mistake caused by the char "+"